src/utils/findPropertySource.ts

```diff
@@ -0,0 +1,12 @@
+export function findPropertySource(
```

Comment:
Could you please add a short description of the intention of this function? So it's clear what it's intended to do as time goes on. Thanks!